StardewArchipelago/Locations/CodeInjections/Vanilla/BackpackInjections.cs

```diff
@@ -47,6 +50,14 @@ public static bool AnswerDialogueAction_BackPackPurchase_Prefix(GameLocation __i
                     return false; // don't run original logic
                 }
 
+                if (_locationChecker.IsLocationNotChecked(PREMIUM_PACK) && Game1.player.Money >= 50000)
+                {
+                    Game1.player.Money -= 10000;

```

Comment:
Nice to have fixed the amount once, but it's there twice.

Maybe this error would have been avoided by using constants (wink wink)